ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -7,7 +7,7 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
-import { A } from '@ember/array';
+import { TrackedArray } from 'tracked-built-ins';
```

Comment:
Why using TrackedArray rather than native JS array? 

[This component implementation is based on trackable array](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/#toc_arrays). When the user click the button to add a worker tag, the `addWorkerTag` function is call, such function pushed the new tag into the array.
`workerConfigText()` is call when the array is updated (because is trackable), then it follows workerConfigText implementation which will call other functions like `getTagConfigString`.

The above is my assumption of this component as specifically `workerTags`. If you think this assumption is wrong, please let me know and I will review this implementation.